### ISSUE TITLE
coninhub components missing for bitgo wallets

### ISSUE THREAD

**Issue by zeus-dev** [openware/opendax#250]
please add coinhub components in opendax and please attach some blockchain examples 

like
coinhub.env
BITGO_ENV: testnet
PORT: 10010

coinhub:
    restart: always
    image: quay.io/openware/coinhub:0.2.8
    env_file:
      - ../config/coinhub.env

**Comment by ultra-pool** [comment]
Yes I would know the same thing about this. And about adding it to the tower.
